docs/api/proposed_endpoints.md

```diff
@@ -356,3 +356,168 @@ Status: 200 OK
     "posted_time": null
 }
 ```
+
+# Talks
+
+## Create Talk
+
+```
+POST /api/talk 
+```
+
+**Parameters**
+
+Name               | Data type     | Description
+-------------------|---------------|---------------------
+id                 | UUID          | Unique ID for the talk
+talk_name          | text          | Name of talk
+speaker            | text          | Speaker for the talk
+type               | choices       | Type like talk, dev sprint, workshop
```

Comment:
Then instead of `talk` the correct word for this API is `proposal`. So the endpoint should be ` /api/proposal` right? And instead of `talk_name` should it be `proposal_description`?
Did I understand it correctly?

Reply:
Yes, it would be `proposal` for this and all the other subsequent APIs.

You don't say `talk_name` or `proposal_description`. You just say `name`, `description` and `abstract`. It is part of `proposal` model which is well evident that `name` is `proposal_name`.

So, no need to prefix your attribute with model name again.